tools/update_versions.py

```diff
@@ -67,6 +67,20 @@ def _get_git_log(repo, since_id):
 
     return log
 
+def _get_git_last_commit_log(repo, last_id):
```

Comment:
I think we could make this function generic. It doesn't need to know that it is using the last commit. Maybe `_get_git_log` or `_get_git_commit_log` and then alter the `last_id` argument to `commit_id`.